examples/logical_interconnect_group.rb

```diff
@@ -12,12 +12,15 @@
 # NOTE: This recipe requires:
 # Ethernet Networks: EthernetNetwork1, EthernetNetwork2
 # FC Network: FCNetwork1
+# Scopes: Scope1, Scope2
 
+# NOTE: The api_version client should be greater than 200 if you run the examples using Scopes
```

Comment:
I'd swap `greater than 200` for `(...) 300 or greater (...)`. Since 200 is not supported, it probably makes sense to not even mention it.